install/kubernetes/cilium/templates/cilium-agent-servicemonitor.yaml

```diff
@@ -36,7 +36,7 @@ metadata:
 spec:
   selector:
     matchLabels:
-      k8s-app: cilium
+      k8s-app: hubble
```

Comment:
It looks like the commit that regressed is actually e9cb43c031, not the PR referenced in the commit. Could you amend the commit to:

```
Fixes: e9cb43c031 ("Helm: full refactor of helm charts, default values implemented, tests updated, kind cni integration")
```

While we're at it, could you amend the commit title to use the [imperative mood](https://chris.beams.io/posts/git-commit/#imperative), so in this case it would be `Fix`.

Reply:
@christarazi I've updated the commit message with the changes.